### ISSUE TITLE
enformer for other species

### ISSUE THREAD

**Issue by yinhongwei4079** [deepmind/deepmind-research#297]
@malcolmreynolds we want to use the enformer for other species like pig. But when we checked the codes of enformer.py and noticed the codes heads_channels = {'human': 5313, 'mouse': 1643}. so we want to know the enformer can be used for other speices ? if we want, need we do something to the codes of enformer.py?

**Comment by amir-azizi** [comment]
Hi Žiga,
Thank you for the response and for the training code. I used it to train Enformer on ATACseq tracks of Rice.

The length of the output tracks in the paper is 896 bins (x128=114,688 bp), but as the GPU I have access to (GeForce RTX 3080) has limited RAM (10 GB), I get OOM errors with this size. Therefore, I reduced the sequence length in the Basenji preproccessing step to 256 (lowest power of 2 that could be handled error free). The training is ongoing now. I was wondering if this size reduction can cause any problems.

**Comment by Avsecz** [comment]
By 256 do you mean 256 bins or bp? I would recommend lowering the number of channels by 4x to say 384 instead of lowering the sequence length. Also, don't forget to compile the model using @tf.function as it will use much less memory and make sure you are using batch size = 1 per GPU.

**Comment by amir-azizi** [comment]
Yes, I meant 256 bins. The predictions were not accurate with that bin-size.
I lowered the number of channels by 4x and the machine can now handle 896-bin sequences :)

Thanks a lot

**Comment by MagdalenaZZ** [comment]
Which training data would you need for another species?
